src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/navigate-modal/NavigateModal.scss

```diff
@@ -0,0 +1,102 @@
+@import 'src/styles/common';
+
+.modalWrapper {
+  background: $light-grey;
+}
+
+.navigateModal {
+  width: 272px;
+}
+
+.browserNavBarControls {
+  margin: 15px 0 0 40px;
+
+  .moveLeftButton,
+  .moveRightButton,
+  .zoomInButton,
+  .zoomOutButton {
+    display: inline-block;
+    margin-right: 35px;
+  }
+
+  .moveLeftButton,
+  .moveRightButton {
+    height: 22px;
+    width: 22px;
+  }
+
+  .zoomInButton,
+  .zoomOutButton {
+    height: 16px;
+    width: 16px;
+  }
+
+  .browserReset {
+    height: var(--browser-reset-button-dimension, 16px);
+    width: var(--browser-reset-button-dimension, 16px);
+  }
+}
+
+.inputGroup {
+  margin-top: 10px;
+  height: 30px;
+  font-weight: $light;
+
+  label {
+    & > span {
+      display: inline-block;
+      margin-right: 10px;
+      text-align: right;
+      width: 32px;
+    }
+  }
+
+  input {
+    border: 1px solid $light-grey;
+    height: 30px;
+    padding: 2px 6px;
+
+    &:disabled {
+      border: 1px solid $grey;
+    }
+  }
```

Comment:
![image](https://user-images.githubusercontent.com/6834224/209310227-d2aee3c5-cb2e-4712-bb2c-9cb39620999e.png)

The styles that we are seeing here should probably be defined on the Input component, and be adjustable through variables. Probably not for this PR; but we should probably have a ticket for this.